source/crud/tests/README.rst

```diff
@@ -82,6 +65,11 @@ Each YAML file has the following keys:
       result object if their BulkWriteException (or equivalent) provides access
       to a write result object.
 
+  - ``expectations`` (optional): List of command-started events.
```

Comment:
This suggests that there is a standard way to describe `command-started events`. Is the expected format of these events that same as that mentioned [in the transactions spec](https://github.com/mongodb/specifications/tree/master/source/transactions/tests#command-started-events)? If so, we should probably make 'command-started events' a link to the same.

Reply:
The original format for these expectations comes from the Command Monitoring spec test. Unfortunately, that spec's [README.md](https://github.com/mongodb/specifications/blob/master/source/command-monitoring/tests/README.rst) also doesn't have a real format defined (like other spec test READMEs) and assumes that the document fields in the YAML/JSON file correspond to the event models from the spec itself.

I'm not sure it makes sense to link to the Transaction spec's README here, as the only thing it says for this field is "The event listener used for these tests MUST ignore the security commands listed in the Command Monitoring Spec." I don't think that's particularly helpful, as the [Security](https://github.com/mongodb/specifications/blob/master/source/command-monitoring/command-monitoring.rst#security) section in the command monitoring spec already discusses such commands being filtered out. AFAIK, libmongoc doesn't even raise events for these commands (vs. using an empty document) -- I'm not sure if other drivers do the same or actually report events w/ empty documents.

Reply:
Yes, I think that will suffice.

Reply:
Updated the format documentation to refer to Command Monitoring spec. Feel free to resolve.